fidimag/atomistic/anisotropy.py

```diff
@@ -65,13 +65,14 @@ def compute_field(self, t=0, spin=None):
 
         clib.compute_anisotropy(m,
                                 self.field,
+                                self.mu_s_inv,
                                 self.energy,
                                 self._Ku,
                                 self._axis,
                                 self.n
                                 )
 
-        return self.field * self.mu_s_inv
+        return self.field
```

Comment:
Why has this needed to change? Not sure I understand

Reply:
I moved the multiplication to the C library, since `mu_s_inv` is an `n`-size array and we needed to create a `3 * n` array to perform the muliplication to the field (at least we reshape the field and multiply by `mu_s_inv[:, np.newaxis]`, which doesn't look that clear in the code)